src/MagnumPlugins/WavAudioImporter/Test/WavImporterTest.cpp

```diff
@@ -278,6 +312,10 @@ void WavImporterTest::mono32f() {
 
     CORRADE_COMPARE(importer.format(), Buffer::Format::MonoFloat);
     CORRADE_COMPARE(importer.frequency(), 48000);
+
+    CORRADE_COMPARE_AS(importer.data().slice(0, 16),
+        Containers::Array<char>::from(0, 0, 0, 0, 108, 57, -103, 59, 3, 63, 42, 60, -33, -81, -120, 60).slice(0, 16),
+        TestSuite::Compare::Container);
```

Comment:
For the 16-bit, 32-bit and 64-bit formats it might make sense to compare the data as actual types and not as their byte representation. That will also account for potential endianness issues. Currently you have to do the casting manually (a helper function is on my TODO list):

```cpp
Containers::Array<char> data = importer.data();
CORRADE_VERIFY(data.size() >= 4*sizeof(Float));
CORRADE_COMPARE(Containers::ArrayView<Float>{reinterpret_cast<Float*>(data.data()), 4},
    Containers::Array<Float>::from(...),
    TestSuite::Compare::Container);
```